parserator_web/static/js/index.js

```diff
@@ -1 +1,92 @@
-/* JS code goes here */
+/* I got a lint error for this async function
+   "Unexpected token function"
+   but wasn't able to fix, and kept function as is */
+async function getAndDisplayAddressData(inputString) {
+  const url = "api/parse/?input_string=" + inputString;
+  const response = await fetch(url, {
+    method: "GET",
+  })
+    .then((response) => response.json())
```

Comment:
since you are in an async function, you can take fuller advantage of awaits, like...

```
response = await fetch(url)
data = await response.data()
...
```

Reply:
Ooh, good point, thank you!